.github/ISSUE_TEMPLATE/2-install-issue.yml

```diff
@@ -42,6 +42,13 @@ body:
         - other
     validations:
       required: true
+  - type: input
+    id: package-manager-version
+    attributes:
+      label: Package Manager Version
+      description: What version of your chosen package manager
```

Comment:
It's minor, but there is some inconsistency in how we're wording these descriptions. For the Cypress Version attribute above, the description is a statement. For this one and the below, it's a question.

For this one in particular, it also seems to be an incomplete thought, though. I'd personally prefer consistency across all the questions, but at a minimum, I think we could fix up the wording on this one.

Reply:
good call out.  I will update